maps/q_boid/map.xml

```diff
@@ -0,0 +1,307 @@
+        <item slot="5" amount="16">vine</item>
+        <potion duration="oo" amplifier="1" ambient="true">fast digging</potion>
+    </kit>
+    <kit name="yellow" parents="spawn">
+        <helmet color="e5e533">leather helmet</helmet>
+        <chestplate color="e5e533">leather chestplate</chestplate>
+        <boots enchantment="protection fall:1" color="e5e533">leather boots</boots>
+        <item slot="2" damage="4" amount="15">stained glass</item>
+        <item slot="3" damage="4" amount="8">wool</item>
+    </kit>
+    <kit name="lime" parents="spawn">
+        <helmet color="33c633">leather helmet</helmet>
+        <chestplate color="33c633">leather chestplate</chestplate>
+        <boots enchantment="protection fall:1" color="33c633">leather boots</boots>
+        <item slot="2" damage="5" amount="15">stained glass</item>
+        <item slot="3" damage="5" amount="8">wool</item>
+    </kit>
+</kits>
+<spawns>
+	<!-- make these points -->
+    <spawn team="yellow" kit="yellow" yaw="0">
+        <cuboid min="69,90,-34" max="66,90,-35"/>
+        <cuboid min="76,83,-34" max="73,83,-35"/>
+        <cuboid min="83,76,-34" max="80,76,-35"/>
+    </spawn>
+    <spawn team="lime" kit="lime" yaw="-90">
+        <cuboid min="17,90,12" max="16,90,15"/>
+        <cuboid min="17,83,19" max="16,83,22"/>
+        <cuboid min="17,76,26" max="16,76,29"/>
+    </spawn>
+    <default yaw="135">
+        <point>85.5,72,31.5</point>
+    </default>
+</spawns>
+<!-- now called kill-rewards -->
+<kill-rewards>
+    <kill-reward>
+        <item amount="5">vine</item>
+        <item amount="8">arrow</item>
+        <item>golden apple</item>
+    </kill-reward>
+    <!-- give TNT to the player every three kills -->
+    <kill-reward>
+        <filter>
+            <kill-streak count="3" repeat="true"/>
+        </filter>
+        <item>tnt</item>
+    </kill-reward>
+    <kill-reward>
+        <filter>
+            <team>yellow</team>
+        </filter>
+        <item amount="10" damage="4">stained glass</item>
+        <item amount="5" damage="4">wool</item>
+    </kill-reward>
+    <kill-reward>
+        <filter>
+            <team>lime</team>
+        </filter>
+        <item amount="10" damage="5">stained glass</item>
+        <item amount="5" damage="5">wool</item>
+    </kill-reward>
+</kill-rewards>
+<toolrepair>
+    <tool>stone sword</tool>
+    <tool>bow</tool>
+    <tool>shears</tool>
+</toolrepair>
+<itemremove>
+    <item>arrow</item>
+    <item>golden carrot</item>
+    <item>leather boots</item>
+    <item>leather chestplate</item>
+    <item>leather helmet</item>
+    <item>leather leggings</item>
+    <item>wool</item>
+</itemremove>
+<tnt>
+    <fuse>3s</fuse>
+    <instantignite>on</instantignite>
+    <power>2</power>
+</tnt>
+<disabledamage>
+    <damage ally="false" self="false" enemy="true" other="false">block explosion</damage>
+</disabledamage>
+<regions>
+    <apply block-break="no-break">
+      <rectangle min="oo,oo" max="oo,oo"/>
```

Comment:
Think you meant to include everywhere right? To fix this if you want, just remove the rectangle and close the <apply>. If no region is specified, it defaults to everywhere

Reply:
Or just do -oo in the min